docs/examples/201/vm-copy-managed-disks/main.bicep

```diff
@@ -0,0 +1,188 @@
+param virtualMachineAdminUserName string = 'azadmin'
+@secure()
```

Comment:
one more nitpick - can we add a blank line above the first decorator for each param so that it is clear which decorator applies to which param?

Reply:
You shouldn't need to regenerate the JSON btw as that will cause an error unless you generate your JSON files with `dotnet test` instead of `bicep build`